wallet-new/src/Cardano/Wallet/WalletLayer/Error.hs

```diff
@@ -13,13 +13,15 @@ import           Universum
 import qualified Data.Text.Buildable
 import           Formatting (bprint, stext, (%))
 
-import           Cardano.Wallet.API.V1.Types (WalletId, AccountIndex)
+import           Cardano.Wallet.API.V1.Types (AccountIndex, WalletId)
 
 
 data WalletLayerError
     = WalletNotFound WalletId
     | AccountNotFound WalletId AccountIndex
     | AddressNotFound WalletId AccountIndex
+    -- NOTE: optionally show mnemonic of the wallet (adn thinks its a bad idea https://iohk.myjetbrains.com/youtrack/issue/CSL-2431#comment=93-19922 )
+    | WalletAlreadyExists
```

Comment:
Hm, unfortunatelly walletId is returned only in case wallet creation/restore was successful. So we don't have this information given us from V0 endpoint in the case of failure :( 

Possible solutions:
 * create an yt issue to add this information in new wallet redesign
     - when we figure out internally there is a walletId that contains same mnemonic we will properly inform the user with the walletId
 * showing backup phrase seems unnecessary (leaking info++) so its not an option